app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split(" ")
+    if len(command) == 3:
+        function, input_file, output_file = command
+        if function != function or input_file == output_file:
```

Comment:
what do you check by this `if function != function`. You should check if command is `cp`